plugins/delphes/DelphesPythia8Reader.h

```diff
@@ -209,12 +209,14 @@ class DelphesPythia8Reader: public DelphesInputReader {
     pythia = new Pythia8::Pythia;
 
     // jet matching
+#if PYTHIA_VERSION_INTEGER < 8300
     matching = combined->getHook(*pythia);
     if(!matching)
     {
-      throw std::runtime_error("can't do matching");
+      throw runtime_error("can't do matching");
```

Comment:
This line trips up the CI. I think the namespace shouldn't be removed.
```suggestion
      throw std::runtime_error("can't do matching");
```

Reply:
yep - copy paste error...